OJP/OJP_JourneySupport.xsd

```diff
@@ -467,7 +477,9 @@
 				</xs:element>
 				<xs:element name="TimeAllowance" type="xs:duration" minOccurs="0">
 					<xs:annotation>
-						<xs:documentation>Extra time needed before reaching/after leaving this place.</xs:documentation>
+						<xs:documentation>Extra time needed before reaching/after leaving this place.
+						
```

Comment:
I'm not sure what you mean by "Overloading the meaning in a responding system is discouraged."

Reply:
Jona: expected time for the whole journey in their responsibility. For the long distance part. We need it somewhere.  Only Andreas Horst or Alexandre.  
Alexandre: We use it as a TimeOffset. n:m Trip Planning.